.github/workflows/opampsupervisor-release.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
       - uses: anchore/sbom-action/download-syft@e11c554f704a0b820cbf8c51673f6945e0731532 # v0.20.0
       - uses: docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392 # v3.6.0
         with:
-          platforms: amd64, arm64,ppc64le
+          platforms: amd64,arm64,ppc64le,riscv64
```

Comment:
this is not needed since the opampsupervisor is not set up with riscv64 in this PR.

you can either remove this here or add riscv64 to the opamp supervisor